src/openms/include/OpenMS/CHEMISTRY/ModificationDefinition.h

```diff
@@ -63,8 +63,11 @@ namespace OpenMS
     /// copy constructor
     ModificationDefinition(const ModificationDefinition& rhs);
 
-    /// detailed constructor specifying the modifications name
-    explicit ModificationDefinition(const String& mod);
+    /// detailed constructor specifying the modification by name
+    explicit ModificationDefinition(const String& mod, bool fixed = true, UInt max_occur = 0);
+
+    /// detailed constructor specifying the modification by name
+    ModificationDefinition(const ResidueModification& mod, bool fixed = true, UInt max_occur = 0);
```

Comment:
Hm. What's the disadvantage of allowing this conversion? Both classes represent pretty much the same thing.